Maple2.File.Parser/TableParser.cs

```diff
@@ -223,7 +234,11 @@ public TableParser(M2dReader xmlReader) {
     }
 
     public IEnumerable<(int Id, ColorPalette Palette)> ParseColorPaletteAchieve() {
-        XmlReader reader = xmlReader.GetXmlReader(xmlReader.GetEntry("table/na/colorpalette_achieve.xml"));
+        string filename = "table/na/colorpalette_achieve.xml";
+        if (locale == "kr") {
+            filename = "table/colorpalette.xml";
```

Comment:
kr doesn't actually have this xml because color palettes locked behind achievements wasn't a KMS2 thing. I think we should just return empty here.

Reply:
oh ok